cmd/gazelle/diff.go

```diff
@@ -27,6 +27,12 @@ import (
 	"github.com/pmezard/go-difflib/difflib"
 )
 
+type exitError struct{}
```

Comment:
An alternative way of doing this would be to explicitly define an error value with something like 

```go
var (
  exitError = fmt.Errorf("Encountered errors while running diff")
)
```